### PR TITLE
Fix tracing span name being empty for pointer types

### PR DESCRIPTION
**Pull request by gnirb**

### CODE REVIEW

File: ops/instrument_pam.go [hunk 1]
```diff
@@ -102,10 +103,20 @@ func PAMTracingHandler(tracerName string, attributes ...attribute.KeyValue) pipe
 	}
 }
 
-// getRequestName, return "GetSession" from "GetBalanceRequest"
+// getRequestName, return "vplugin.PluginPAM/GetSession" from "GetBalanceRequest"
```
Comment (gustavste) [inline ops/instrument_pam.go:106]: Is it suppose to be `"...GetBalance" from "GetBalanceRequest"`?

Comment (gnirb) [reply ops/instrument_pam.go:106]: Yep.. Totally missed that they didn't match 😆

File: ops/instrument_pam.go [hunk 2]
```diff
@@ -102,10 +103,20 @@ func PAMTracingHandler(tracerName string, attributes ...attribute.KeyValue) pipe
...
+	t := reflect.TypeOf(req)
+	if t.Kind() == reflect.Pointer {
+		name = t.Elem().Name()
+	} else {
+		name = t.Name()
+	}
+
+	name, _ = strings.CutSuffix(name, "Request")
+
+	return fmt.Sprintf("%s/%s", RPCService, name)
```
Comment (gustavste) [inline ops/instrument_pam.go:119]: Isn't this method used both when using genericpam and vplugin. in the `PAMMetricHandler`. Adding "vplugin.PluginPAM" doesn't seem correct in the generic case

Comment (gnirb) [reply ops/instrument_pam.go:119]: Right you are again! 😆 Let me fix that!

**Review by gustavste — APPROVED** [review]
👍

**Pull request merged**